website/docs/gs/input_files.md

```diff
@@ -5,11 +5,88 @@ sidebar_position: 3
 slug: ./inputs
 ---
 
-GATK-SV supports input data in the following formats.
+GATK-SV requires the following input data:
 
 - Illumina short-read whole-genome CRAMs or BAMs, aligned to hg38 with [bwa-mem](https://github.com/lh3/bwa). 
   BAMs must also be indexed.
 
 - Family structure definitions file in 
   [PED format](https://gatk.broadinstitute.org/hc/en-us/articles/360035531972-PED-Pedigree-format).  
   **Note**: Sex aneuploidies (detected in EvidenceQC) should be entered as sex = 0.
+
+### Sample Exclusion
+We recommend filtering out samples with a high percentage 
+of improperly paired reads (>10% or an outlier for your data) 
+as technical outliers prior to running [GatherSampleEvidence](/docs/modules/gse). 
+A high percentage of improperly paired reads may indicate issues 
+with library prep, degradation, or contamination. Artifactual 
+improperly paired reads could cause incorrect SV calls, and 
+these samples have been observed to have longer runtimes and 
+higher compute costs for [GatherSampleEvidence](/docs/modules/gse).
+
+
+### Sample ID requirements
+#### Sample IDs must
+
+- Be unique within the cohort
+- Contain only alphanumeric characters and underscores (no dashes, whitespace, or special characters)
+
+Sample IDs should not:
```

Comment:
I think this format should match with the format for "Sample IDs must"

Reply:
I suppose you mean the heading level; if so, I fixed it.